index.html

```diff
@@ -1,5 +1,7 @@
+<!doctype html>
```

Comment:
Is `doctype` [supposed to be in all capitals](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/HTML5/Introduction_to_HTML5)?